docs/ota-client-guide/modules/ROOT/pages/sign-with-external-signature.adoc

```diff
@@ -0,0 +1,110 @@
+= Sign metadata with externally generated signature
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+For safety reasons you may want to sign your metadata with the signature generated by your own PKI.
+For more information on where to find your metadata files see the xref:finding-unsigned-metadata.adoc[Find the unsigned Root and Targets metadata].
+
+=== Signing Root metadata
+
+To sign `root.json` with your own generated signature you will need the name of key with which it was signed previously.
+You can use multiple signatures for signing.
+For verification purposes, with each signature provided you will need to specify the name of the key with which it was generated.
+
+```
+garage-sign root sign \
+        --repo <repository-name> \
+        --old-root-alias <key-name> \
+        --signatures <key-name=signature>
+```
+
+=== Signing Targets metadata
+
+For signing `targets.json` the procedure is similar except you don't need the old key name.
+
+NOTE: If you're passing a signature you cannot pass a version.
+
+```
+garage-sign targets sign \
+        --repo <repository-name> \
+        --signatures <key-name=signature>
+```
+
+== Example script with a flow that is using OpenSSL
+
+```
+#!/bin/bash
+
+set -x
+set -euo pipefail
+
+garage_sign="${1}"
```

Comment:
I would get rid of this parameter and call `garage-sign` directly below for the example so make it easier to read. Maybe also get rid of the next parameter.